spec/features/login_page_spec.rb

```diff
@@ -0,0 +1,98 @@
+describe 'login_page' do
+  before { visit '/login' }
+
+  context 'Visibility' do
+    it 'has Email field' do
+      expect(page).to have_css('input#spree_user_email')
+    end
+
+    it 'has Password field' do
+      expect(page).to have_css('input#spree_user_password')
+    end
+
+    it 'has Remember_me checkbox' do
+      expect(page).to have_css('input#spree_user_remember_me')
+      expect(page).to have_css('label', text: 'Remember me')
+    end
+  end
+
+  context 'Login process' do
+    it 'can Log in as a User' do
+      fill_in 'spree_user_email', with: ENV['USERNAME_SPREE']
+      fill_in 'spree_user_password', with: ENV['PASSWORD_SPREE']
```

Comment:
Try separating blocks of code with empty lines for better readability
For instance - you can split this code into 3 sections
1. Filling inputs
2. Clicking the button
3. Expectations

Based on that it'd be nice to add empty line after lines 22 & 23 🙂 
Same goes for other specs